### ISSUE TITLE
[Dataset update] Paper 527: doi, publisher_url, venue, access, bibliographic

### ISSUE THREAD

**Issue by ahafuaej-alt** [ahafuaej-alt/PINN-Review#168]
## PINN Review Atlas dataset update

- **Paper:** [527] Dual Cone Gradient Descent for Training Physics-Informed Neural Networks
- **Changed fields:** doi, publisher_url, venue, access, bibliographic
- **Citation mode:** automatic
- **Evidence:** https://proceedings.neurips.cc/paper_files/paper/2024/file/b2b781badeeb49896c4b324c466ec442-Paper-Conference.pdf

### Reason
all info

### MDPI citation preview
> Hwang, Y., Lim, D.-Y. Dual Cone Gradient Descent for Training Physics-Informed Neural Networks. In Advances in Neural Information Processing Systems; Globerson, A., Mackey, L., Belgrave, D., Fan, A., Paquet, U., Tomczak, J., Zhang, C, Eds.; Curran Associates, Inc: Ulsan, South Korea, 2024; Volume 37, pp. 98563-98595. https://doi.org/10.52202/079017-3128.

### Maintainer workflow
The repository will validate this request automatically. After checking the evidence and validation comment, a maintainer can apply it by commenting `/apply-dataset-update`.
